modules/Hoot/ui/conflicts/info.js

```diff
@@ -84,18 +85,43 @@ export default class ConflictMetadata {
                 .classed( 'value-col feature1', true )
                 .text( tag.value[ 0 ] );
 
+            row.selectAll( 'td.value-col.feature1' )
+                .on('click', () => {
+                    this.panToEntity(this.data.currentFeatures[0]);
+                    this.selectEntity(this.data.currentFeatures[0]);
+                });
+
             row.selectAll( 'td.feature2' )
                 .data( [ { k: 2 } ] ).enter()
                 .append( 'td' )
                 .classed( 'value-col feature2', true )
                 .text( tag.value[ 1 ] );
+
+            row.selectAll( 'td.value-col.feature2' )
+                .on('click', () => {
+                    this.panToEntity(this.data.currentFeatures[1]);
+                    this.selectEntity(this.data.currentFeatures[1]);
+                });
         } );
 
         this.poiTable.selectAll( '.value-col' )
             .on( 'mouseenter', d => d3.selectAll( `.review-feature${ d.k }` ).classed( 'extra-highlight', true ) )
             .on( 'mouseleave', d => d3.selectAll( `.review-feature${ d.k }` ).classed( 'extra-highlight', false ) );
     }
 
+    selectEntity(entity) {
+        Hoot.context.enter(modeSelect(Hoot.context, [entity.id]));
+    }
+
+    /**
+     * Pan to feature with conflict
+     */
+
+    panToEntity(feature) {
+        let extent = feature.extent(Hoot.context.graph());
+        Hoot.context.map().centerZoom(extent.center(), 21);
```

Comment:
After merging hoot2x into this branch, there is [this](https://github.com/ngageoint/hootenanny-ui/blob/cf3e5b09633fd5cd698f21f11c059281008ffacf/modules/Hoot/ui/conflicts/map.js) function on the conflict's Map class.
We should use it here too.
However, I think we'll need to make it a static method so we can use it here without make a new instance of the class.